Application/Telegram/Keyboards/CommandsListKeyboard.cs

```diff
@@ -14,5 +15,12 @@ public static InlineKeyboardMarkup BuildKeyboard(List<string> commandNames, int
             var inlineKeyboard = KeyboardHelper.BuildInLineKeyboard(buttons, columns);
             return inlineKeyboard;
         }
+        public static InlineKeyboardMarkup BuildKeyboard(List<string> commandNames, int columns, UserState currentState)
+        {
+            commandNames.Add("◀️Go Back");
+            var inlineKeyboard = CommandsListKeyboard.BuildKeyboard(commandNames, columns);
+            inlineKeyboard.InlineKeyboard.ToList().Last().ToList().Last().CallbackData = $"goBack|{currentState.ToString()}";
```

Comment:
how can simplify this call ? or should I change BuildKeyboard function and should make it like that?

public static InlineKeyboardMarkup BuildKeyboard(List<string> commandNames, int columns, string stateName = "defaultState")
        {
            var buttons = new List<InlineKeyboardButton>();
            foreach (var name in commandNames)
            {
                buttons.Add(new InlineKeyboardButton(name) { CallbackData = name});
            }
            buttons.Add(new InlineKeyboardButton($"◀️Go Back") { CallbackData = $"goBack|{stateName}"});
            var inlineKeyboard = KeyboardHelper.BuildInLineKeyboard(buttons, columns);
            return inlineKeyboard;
        }

Reply:
yes, we should change, right now it's non-readable, and in your comment is better. and maybe add some flag like "includeBackButton) - if this button would not on each page